docs/src/main/tut/faq.md

```diff
@@ -16,7 +16,7 @@ section: "faq"
  * [What does `macro Ops` do? What is `cats.macros.Ops`?](#machinist)
  * [How can I help?](#contributing)
 
-## What imports do I need?<a id="what-imports" href="#what-imports"></a>
+## <a id="what-imports" href="#what-imports"></a>What imports do I need?
```

Comment:
I put the links at the end of the header because I added a bit of CSS that makes an anchor pop up with a link for each header and if it's on the left side, the header text shifts around which is a bit disorienting. This is something that isn't noticeable if you are just looking at the markdown view, but it will be noticeable once the FAQ is deployed on the site.

I'm definitely open to better options -- I was just looking for low-hanging fruit for easy FAQ links that I could implement with very limited CSS skills :)